frontend/src/component/project/ProjectCard/ProjectCard.tsx

```diff
@@ -50,118 +45,90 @@ export const ProjectCard = ({
-                        event.preventDefault();
-                        setAnchorEl(null);
-                    }}
-                >
-                    <MenuItem
-                        onClick={(e) => {
-                            e.preventDefault();
-                            navigate(getProjectEditPath(id));
+                    <Menu
+                        id='project-card-menu'
+                        open={Boolean(anchorEl)}
+                        anchorEl={anchorEl}
+                        style={{ top: 0, left: -100 }}
+                        onClick={(event) => {
+                            event.preventDefault();
+                        }}
+                        onClose={(event: SyntheticEvent) => {
+                            event.preventDefault();
+                            setAnchorEl(null);
                         }}
                     >
-                        <StyledEditIcon />
-                        Edit project
-                    </MenuItem>
-                </Menu>
-            </StyledDivHeader>
-            <StyledIconBox data-loading>
-                <ConditionallyRender
-                    condition={mode === 'private'}
-                    show={<HiddenProjectIconWithTooltip />}
-                    elseShow={<StyledProjectIcon />}
-                />
-            </StyledIconBox>
-            <StyledDivInfo>
-                <StyledDivInfoContainer>
-                    <StyledParagraphInfo data-loading>
-                        {featureCount}
-                    </StyledParagraphInfo>
-                    <p data-loading>toggles</p>
-                </StyledDivInfoContainer>
-                <StyledDivInfoContainer>
-                    <StyledParagraphInfo data-loading>
-                        {health}%
-                    </StyledParagraphInfo>
-                    <p data-loading>health</p>
-                </StyledDivInfoContainer>
+                        <MenuItem
+                            onClick={(e) => {
+                                e.preventDefault();
+                                navigate(getProjectEditPath(id));
+                            }}
+                        >
+                            <StyledEditIcon />
+                            Edit project
+                        </MenuItem>
+                    </Menu>
+                </StyledDivHeader>
+                <StyledDivInfo>
+                    <StyledDivInfoContainer>
+                        <StyledParagraphInfo data-loading>
+                            {featureCount}
+                        </StyledParagraphInfo>
+                        <p data-loading>toggles</p>
+                    </StyledDivInfoContainer>
+                    <StyledDivInfoContainer>
+                        <StyledParagraphInfo data-loading>
+                            {health}%
+                        </StyledParagraphInfo>
+                        <p data-loading>health</p>
```

Comment:
Should be one paragraph. The number can be in a span if it needs to be styled differently.